src/main/resources/application.conf

```diff
@@ -33,19 +33,19 @@ IkeToolWebapp = {
     //  location = file
     //  description = "My Blacklab-indexed corpus"
     //  path = /path/to/index-directory
-    //} 
+    //}
   ]
 }
 
 Tablestore = {
   db = {
-    // NOTE: Before running locally, uncomment the below lines and specify values for keys `url` to point to your PostgresSql JDBC 
+    // NOTE: Before running locally, uncomment the below lines and specify values for keys `url` to point to your PostgresSql JDBC
     // link, `user` and `password`.
     // AI2 Internal users: use the database in the `test` deploy environment and the same username. You can get the password
     // from this file in the ops-keystore in S3.
-    // url =
-    // user =
-    // password =
+    url = "jdbc:postgresql://"${POSTGRES_DB}":5432/okcorpus"
+    user = "okcorpus"
+    password = ${POSTGRES_PASSWORD}
```

Comment:
Why did you remove this? Can you still deploy to the IKE production server without this?